tools/scope_upload_classification.py

```diff
@@ -35,6 +39,12 @@ def upload_classification(
     sources = pd.read_csv(file)
     columns = sources.columns
 
+    if start is not None:
+        if stop is not None:
+            sources = sources.loc[start:stop]
```

Comment:
Should this be start+1?

Reply:
Since the code currently prints the zero-based index of each row in the dataframe, I wanted to stick with that convention for the start argument. I've noted this convention in usage.md and the help info of the scripts.